### ISSUE TITLE
Browser window is not activated after clicking link

### ISSUE THREAD

**Issue by marcelkorpel** [bbidulock/icewm#196]
When I click a link in one application, e.g. Thunderbird or urxvt, the link is sent to my default browser (in my case Firefox). However, the browser window is not activated, as it was in 1.3.8.

**Comment by gijsbers** [comment]
Is there any effect? What proves that this is caused by IceWM?

**Comment by marcelkorpel** [comment]
This behaviour only popped up after installing IceWM from Github. But it is rather inpredictable, as I just clicked on the link to this issue in an email in Thunderbird and now the Firefox window is raised, as expected.

Couldn't it be related to focus (though I don't think so)? I'm now in 'quiet sloppy' mode (which is new to me) instead of 'sloppy'. I also can't find it in the manual in `/usr/share/doc/icewm/icewm.htm`.

**Comment by marcelkorpel** [comment]
And the effect is that the actual link is opened in a new tab in my default browser, only the window is not raised.

**Comment by gijsbers** [comment]
If you examine the output of `xprop` for the browser when it is raised and when it is not do you see differences? Try multiple times to get a consistent measurement.

**Comment by marcelkorpel** [comment]
I just had a whole series of non-raised and (after logging out and in again) raised windows after clicking on a link. I don't see any clear differences (apart from the obvious ones), only that `_NET_WM_STATE(ATOM)` is output earlier when a window is raised.

[diff.txt](https://github.com/bbidulock/icewm/files/1476367/diff.txt)

**Comment by gijsbers** [comment]
Does it help if you set `RaiseOnFocus=1` ?

**Comment by marcelkorpel** [comment]
That is already the case. Strangely enough, I haven't seen this bug, the last days.

**Comment by gijsbers** [comment]
@marcelkorpel Did you ever see this again? And/or recently?

**Comment by marcelkorpel** [comment]
Yes, every now and then, alas. The last time was about two weeks ago.
